opencti-platform/opencti-front/src/private/components/common/form/GroupField.jsx

```diff
@@ -53,18 +53,22 @@ class GroupField extends Component {
   }
 
   searchGroups() {
-    fetchQuery(groupsQuery)
-      .toPromise()
-      .then((data) => {
-        const groups = pipe(
-          pathOr([], ['groups', 'edges']),
-          map((n) => ({
-            label: n.node.name,
-            value: n.node.id,
-          })),
-        )(data);
-        this.setState({ groups });
-      });
+    if (this.props.predefinedGroups) {
+      this.setState({ groups: this.props.predefinedGroups });
```

Comment:
does it mean that there is no search when we have predefinedGroups ? it will always display the same list ?

Reply:
Yep. You can still search but withing the pre defined list of groups